client/post-editor/editor-term-selector/index.jsx

```diff
@@ -68,23 +81,24 @@ class EditorTermSelector extends Component {
 					siteId={ siteId }
 					multiple={ true }
 				/>
-				<AddTerm taxonomy={ taxonomyName } postType={ postType } />
+				<AddTerm taxonomy={ taxonomyName } postType={ postType } postId={ postId } />
```

Comment:
I don't think it's extreme enough yet to warrant it, and perhaps a bit magical, but at a certain point you could consider using object shorthand spread props:

```jsx
<AddTerm { ...{ taxonomyName, postType, postId } } />
```